middleware/static.go

```diff
@@ -167,7 +177,7 @@ func StaticWithConfig(config StaticConfig) echo.MiddlewareFunc {
 			if err != nil {
 				return
 			}
-			name := filepath.Join(config.Root, filepath.Clean("/"+p)) // "/"+ for security
+			name := path.Join(config.Root, path.Clean("/"+p)) // "/"+ for security
```

Comment:
regarding to `filepath.Clean` vs `path.Clean` there is subtle difference in their behavior https://github.com/labstack/echo/pull/1718#issuecomment-744048995